CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
+#### Introduction
+
+The Gonum project provides general purpose numerical computing and data science libraries for the Go programming language, and we would like you to join us in improving Gonum's quality and scope.
+This document is for anyone who is contributing or interested in contributing.
+
+#### Table of Contents
+
+[Getting Started](#getting-started)
+  * [Working Together](#working-together)
+  * [Project Goals](#project-goals)
+  * [The Gonum Repositories](#the-gonum-repositories)
+
+[Contributing](#Contributing)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Code Contribution](#code-contribution)
+  * [Code Review](#code-review)
+  * [What Can I Do to Help?](#what-can-i-do-to-help)
+  * [Style](#style)
+
+## Getting Started
```

Comment:
I adapted the structure from atom's [contribuing.md](https://github.com/atom/atom/blob/master/CONTRIBUTING.md) - they call it "What should I know before I get started?"

In ours the first part covers basically "what", and the contributing section is "how."   How about moving the Working Together subsection to the top of the Contributing section, and making the "Project Goals" and "The Gonum Repositories" top level sections instead of subsections?